manifests/install/charts/as-a-second-scheduler/templates/configmap.yaml

```diff
@@ -10,8 +10,10 @@ data:
     leaderElection:
       leaderElect: false
     profiles:
+    # user can add as many profiles here. Adding two profiles as an example
```

Comment:
The current templating needs some improvement. I played with Golang/Helm templating a bit, here is my suggestion:

```yaml
---
plugins:
{{- /* Ensure only the first queueSort plugin is rendered. */}}
{{- /* Note: the trick is b/c no "break" support in Go templating. */}}

{{- $queueSortFound := false }}
{{- range $p := .Values.plugins }}
{{- if and (has (title $p) $.Values.queueSort) (not $queueSortFound) }}
{{- $queueSortFound =  true }}
queueSort:
  enabled:
  - name: {{ title $p }}
  disabled:
  - name: "*"
{{- end }}
{{- end }}

{{- range $extension, $plugins := .Values.extensions }}
{{- $pluginFound := false }}
  {{- range $p := $.Values.plugins }}
  {{- if has (title $p) $plugins }}
{{- if not $pluginFound }}
{{ $extension }}:
  enabled:
{{- end }}
  - name: {{ title $p }}
  {{- $pluginFound = true }}
  {{- end }}
  {{- end }}
{{- end }}

{{- /* TODO: support pluginConfig. */}}
{{- /* TODO: tire CRD installation with enabled plugins. */}}
```

And the `values.yaml` can be like this:

```yaml
---
plugins: [""]

# queueSort is not indented under extensions
# as it needs to be globally enabled once.
queueSort: ["Coscheduling"]

extensions:
  preFilter: ["Coscheduling", "CapacityScheduling"]
  filter: ["NodeResourceTopologyMatch"]
  postFilter: ["Coscheduling", "CapacityScheduling", "PreemptionToleration"]
  score: ["NodeResourceTopologyMatch", "LoadVariationRiskBalancing", "TargetLoadPacking", "NodeResourcesAllocatable"]
  permit: ["Coscheduling"]
  reserve: ["Coscheduling", "CapacityScheduling"]
  postBind: ["Coscheduling"]
```